docs/src/pages/components/cards/NotificationCard.js

```diff
@@ -0,0 +1,130 @@
+import Typography from '@material-ui/core/Typography';
+
+export default function NotificationCard() {
+  const globalTheme = useTheme();
+  const mode = globalTheme.palette.mode;
+  const theme = React.useMemo(
+    () =>
+      createTheme({
+        palette: {
+          mode,
+          ...(mode === 'light' && {
+            text: {
+              primary: '#3D4752',
+              secondary: '#8796A5',
+            },
+          }),
+          divider: mode === 'dark' ? '#132F4C' : '#E5E8EC',
+          success: {
+            main: '#1DB45A',
+          },
+          background: {
+            paper: mode === 'dark' ? '#003A75' : '#fff',
+          },
+        },
+        shape: {
+          borderRadius: 8,
+        },
+        spacing: 10,
+        typography: {
+          fontFamily: [
+            '"PlusJakartaSans"',
+            '-apple-system',
+            'BlinkMacSystemFont',
+            'sans-serif',
+          ].join(','),
+          fontWeightRegular: 500,
+        },
+        components: {
+          MuiAvatar: {
+            styleOverrides: {
+              root: {
+                width: 40,
+                height: 40,
+              },
+            },
+          },
+          MuiButtonBase: {
+            defaultProps: {
+              disableTouchRipple: true,
+            },
+          },
+          MuiChip: {
+            variants: [
+              {
+                props: { variant: 'notification' },
+                style: { color: '#fff', fontSize: 12, height: 18 },
+              },
+            ],
+          },
+          MuiLinearProgress: {
+            styleOverrides: {
+              root: {
+                borderRadius: 10,
+                backgroundColor: 'rgba(255,255,255,0.12)',
+              },
+              bar: {
+                borderRadius: 10,
+                backgroundColor: '#fff',
+              },
+            },
+          },
+          MuiIconButton: {
+            defaultProps: {
+              size: 'small',
+            },
+            styleOverrides: {
+              root: {
+                border: '1px solid #E5E8EC',
+              },
+            },
+          },
+          MuiSvgIcon: {
+            defaultProps: {
+              fontSize: 'small',
+            },
+          },
+        },
+      }),
+    [mode],
+  );
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Card variant="outlined" sx={{ display: 'flex', p: 1.5, maxWidth: 283 }}>
+        <Avatar src="/static/images/avatar/3.jpg" variant="rounded" />
+        <Box ml={1} flexBasis={180} flexGrow={1} minWidth="0px">
+          <Box display="flex" justifyContent="space-between">
+            <Typography color="text.secondary" variant="caption">
+              Angela Erickson
+            </Typography>
+            <Typography color="text.secondary" variant="caption">
+              12:50
+            </Typography>
+          </Box>
+          <Box display="flex" mt={0.5}>
```

Comment:
Does it mark a new direction toward using props first before sx?

Reply:
Not sure. Given that we publicly expose props and sx, it is convenient for me to not use `sx` in `Box`. I will use `sx` when I want to override nested component or do something with pseudo class.

The question is does it really matter if I use `props` or `sx` in the docs?

Reply:
Looking back, lean toward `sx` might be better for readability because it reflect that the value will associate with system. will fix all of them.

Reply:
> The question is does it really matter if I use props or sx in the docs?

I think it does yes. Developers are looking for guidance on the documentation, on the best practices. Here, the question is about either it's OK to recommend the flatten props, at the cost of also having sx as soon as a CSS property is not supported and the developer is too lazy to refactor the previous props.